python/lsst/ts/externalscripts/maintel/take_twilight_flats_lsstcam.py

```diff
@@ -223,3 +224,21 @@ async def slew_azel_and_setup_instrument(self, az, el):
             az=az,
             el=el,
         )
+
+    async def configure(self, config):
+        """Take the sequence of twilight flats twilight flats."""
+        await super().configure(config)
+
+        if hasattr(config, "ignore"):
+            # Ignoring only specific MTCS components
+            allowed_ignore = ["mtdome", "mtdometrajectory"]
+            for comp in map(RemoteGroup._remote_name_to_attr_format, config.ignore):
```

Comment:
Again, this is using a "private" method from `RemoteGroup`. Not sure I understand what is going on here. Can you please, clarify?